.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     # these tests based on the nightly compiler, which can be broken
     # consequently, we continue on an error
-    continue-on-error: true
+    #continue-on-error: true
```

Comment:
Why did you comment this?

Reply:
I want to stop the pipeline, if an error occurs. This was only a workaround, because the previous compiler was not compatible with the current version of libhermit

Reply:
Is it intended to be permanently removed?